xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -1090,7 +1119,9 @@ tsl::Status ROCMBlas::DoBlasGemmStridedBatched(
     const DeviceMemoryBase &a, int lda, int64_t stride_a,
     const DeviceMemoryBase &b, int ldb, int64_t stride_b, const void *beta,
     DeviceMemoryBase *c, int ldc, int64_t stride_c, int batch_count,
-    const NumericOptions &numeric_options) {
+    const NumericOptions &numeric_options,
+    blas::CallContext context) {
```

Comment:
nit: I think this can go into the previous line as well without exceeding the 80 character limit

Reply:
Done